src/Apps/ArtQuiz/Components/ArtQuizMain.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  ArrowLeftIcon,
+  Box,
+  Clickable,
+  CloseIcon,
+  Flex,
+  FullBleed,
+  HeartIcon,
+  Text,
+} from "@artsy/palette"
+import { useArtQuizContext } from "Apps/ArtQuiz/ArtQuizContext"
+import { FC } from "react"
+import { RouterLink } from "System/Router/RouterLink"
+
+interface ArtQuizMainProps {}
+
+export const ArtQuizMain: FC<ArtQuizMainProps> = () => {
+  const { total, index, onNext, onPrevious } = useArtQuizContext()
```

Comment:
It occurs to me now that we don't even need a context? All the state is isolated in this lone component. Will just remove. If we need to lift state, we can always do that.